README.md

```diff
@@ -1,2 +1,28 @@
 # table_to_yaml_pyscripts
-Python scripts to convert diag_table and data_table to yaml format
+Python scripts to convert diag_tables, data tables, and field tables to yaml format
+
+## How to install with pip
+
+1. Create virtual python environment
+
+```
+python3 -m venv /path/to/your/install
+```
+
+2. Activate the environment
+
+```
+source /path/to/your/install/bin/activate
```

Comment:
I think this is different depending on whether you are using bash or csh

Reply:
You're right, it's `activate.csh` for cshell and `activate` for sh/bash. Since sh/bash is more standard, I was thinking cshell users can see the syntax error and then use the cshell version. But it would be clearer to just mention both.

Reply:
Whoops, sorry for leaving this PR hanging on this. I listed both the csh and bash activate scripts, and also tested it just now with your combining branch.